common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -1724,8 +1724,11 @@ def test_all_response_types(self, response_tag):
         self.assertEquals(descriptor.problem_types, {response_tag})
         self.assertEquals(descriptor.index_dictionary(), {
             'content_type': CapaDescriptor.INDEX_CONTENT_TYPE,
-            'display_name': name,
-            'problem_types': [response_tag]
+            'problem_types': [response_tag],
+            'content': {
+                'display_name': name,
+                'capa_content': ''
+            }
```

Comment:
I'm uncertain what consumes the `index_dictionary()` - but has the data schema change been accounted for by the consumer?